WEB_SERVICE/WEB/_SECURE/SECURE_HTTP_HEADER/description.md

```diff
@@ -0,0 +1,12 @@
+Secure Header Setting:
+
+* Content Security Policy
+* Cookie
+* Cross-Origin Resource Sharing
+* HTTP Public Key Pinning
+* Redirection
```

Comment:
What do you mean by secure http header : like for example Redirection what do you mean by it ?

Reply:
I guess it's about HTTP status codes that tell the browser to navigate to a different URL. Just make sure to highlight the risks of open redirects and the necessity of validating redirect targets.